src/app/onion/dashboard/dashboard.component.ts

```diff
@@ -2,28 +2,50 @@ import { Component, OnInit } from '@angular/core';
 import { HistoryService } from 'app/core/history.service';
 import { NavigationEnd, Router } from '@angular/router';
 import { NavbarService } from 'app/core/navbar.service';
+import { LearningObject } from '@entity';
+import { LearningObjectService } from 'app/onion/core/learning-object.service';
+import { AuthService } from 'app/core/auth.service';
 import { Subject } from 'rxjs';
 
 @Component({
   selector: 'clark-dashboard',
   templateUrl: './dashboard.component.html',
-  styleUrls: ['./dashboard.component.scss'],
+  styleUrls: ['./dashboard.component.scss']
 })
-export class DashboardComponent {
+export class DashboardComponent implements OnInit {
   lastLocation: NavigationEnd;
   activeIndex = 0;
+  loading: boolean;
+  releasedLearningObjects: LearningObject[];
+  workingLearningObjects: LearningObject[];
 
   action$: Subject<number> = new Subject();
 
+  filters: any;
```

Comment:
Is there any chance we can type this parameter so that we know the different kinds of values it can have?

Reply:
I changed it to an object an added a comment on the structure